internal/kibana/dashboards.go

```diff
@@ -0,0 +1,75 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package kibana
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/common"
+	"github.com/elastic/elastic-package/internal/logger"
+	"github.com/elastic/elastic-package/internal/multierror"
+)
+
+type exportedType struct {
+	Objects []common.MapStr `json:"objects"`
+}
+
+// Export method exports selected dashboards using the Kibana Export API.
+func (c *Client) Export(dashboardIDs []string) ([]common.MapStr, error) {
+	logger.Debug("Export dashboards using the Kibana Export API")
+
+	var query strings.Builder
+	query.WriteByte('?')
+	for _, dashboardID := range dashboardIDs {
+		query.WriteString("dashboard=")
+		query.WriteString(dashboardID)
+		query.WriteByte('&')
```

Comment:
Not a huge deal but wouldn't you end up with a trailing `&`?

Reply:
Yes, but it doesn't mean in terms of processing.